rfc/src/mdn.mdown

```diff
@@ -0,0 +1,35 @@
+%%%
+    title = "Sending an MDN Response with JMAP"
+    abbrev = "JMAP MDN handling"
+    category = "std"
+    docName = "draft-ietf-jmap-mdn-00"
+    ipr= "trust200902"
+    area = "Applications"
+    workgroup = "JMAP"
+    keyword = ["JMAP", "JSON", "email", "MDN"]
+
+    date = 2018-07-25T00:00:00Z
+
+    [[author]]
+    initials="R.O."
+    surname="Ouazana"
+    fullname="Raphaël Ouazana"
+    role="editor"
+    organization = "Linagora"
+        [author.address]
+        email = "rouazana@linagora.com"
+        uri = "https://www.linagora.com"
+        [author.address.postal]
+            street = "100 Terrasse Boieldieu – Tour Franklin"
+            city = "Paris - La Défense CEDEX"
+            code = "92042"
+            country = "France"
+%%%
+
+.# Abstract
+
+This document specifies a data model for handling [@!RFC8098] MDN messages with a server using JMAP.
```

Comment:
s/a data/the data

Reply:
copy/paster from other jmap spec, so I prefer to keep it like this